pkgs/development/python-modules/doitlive/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, buildPythonApplication, fetchPypi,
+ click, ipython
+}:
+
+buildPythonApplication rec {
+  pname = "doitlive";
+  name = "${pname}-${version}";
+  version = "3.0.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "19i16ca835rb3gal1sxyvpyilj9a80n6nikf0smlzmxck38x86fj";
+  };
+
+  propagatedBuildInputs = [ click ];
+
+  checkInputs = [ ipython ];
```

Comment:
Did you try specifying a custom `checkPhase` such that the tests are run? This should be something like `invoke test` oder `py.test`.

Reply:
I tried, basically using
```
checkInputs = with python3Packages; [ pkgs.git ipython pytest ];
LC_ALL="en_US.UTF-8";
checkPhase = "pytest";
```
Still something like 20 tests were failing due to issues with ipython. Unfortunately could not get it to work. How important is the check phase for python packages?

Reply:
Okay, let's keep the tests disabled. Please add a comment that there were too many failures or something like that.
However, tests are nice to have for automatic updates.